.rubocop.yml

```diff
@@ -12,6 +12,8 @@ AllCops:
     - 'tmp/**/*'
     - 'config/initializers/new_framework_defaults.rb'
     - 'bin/*'
+    - 'db/schema.rb'
+    - 'db/migrate/20161217105518_devise_create_users.rb'
```

Comment:
please remove migration from ignore
What kind of errors you have in 20161217105518_devise_create_users.rb ?